src/Semantics.js

```diff
@@ -451,6 +459,14 @@ Semantics.createSemantics = function(grammar, optSuperSemantics) {
     s.extendOperationOrAttribute.call(s, 'attribute', name, actionDict);
     return proxy;
   };
+  proxy.getActionDict = function(operationOrAttributeName) {
+    var action = s.operations[operationOrAttributeName] || s.attributes[operationOrAttributeName];
+    if (!action) {
+      throw new Error('There is no action has name ' + operationOrAttributeName +
```

Comment:
How about `operationOrAttributeName + ' is not a valid operation or attribute name in this semantics'`?